app/util/im2db_utils.py

```diff
@@ -0,0 +1,165 @@
+import os
```

Comment:
created a separate file to collect utility functions around the interface between image files and storing them in the filesystem and the database